ocs_ci/ocs/elasticsearch.py

```diff
@@ -308,22 +334,35 @@ def get_password(self):
 
     def cleanup(self):
         """
-        Cleanup the environment from all Elasticsearch components, and from the
-        port forwarding process.
+        Cleanup the environment from all Elasticsearch components.
 
         """
         log.info("Teardown the Elasticsearch environment")
         log.info("Deleting all resources")
-        log.info("Deleting the dumper client pod")
-        self.ocp.delete(yaml_file=self.dumper_file)
-        log.info("Deleting the es resource")
-        self.ocp.delete(yaml_file=self.crd)
+        try:
+            log.info("Deleting the dumper client pod")
+            self.ocp.delete(yaml_file=self.dumper_file)
+        except CommandFailed:
+            # in case of the es-dumper did not deployed yet, trying to delete it
+            # will failed.
+            pass
```

Comment:
Good to add a log.warn message that expected pod is not deployed so skipped deleting.